src/main/java/it/pagopa/pm/gateway/client/util/ClientConfig.java

```diff
@@ -1,13 +1,12 @@
 package it.pagopa.pm.gateway.client.util;
 
-import it.pagopa.pm.gateway.client.payment.gateway.client.BancomatPayClientV2;
-import it.pagopa.pm.gateway.client.restapiCD.RestapiCdClient;
+import it.pagopa.pm.gateway.client.bpay.*;
```

Comment:
there are differing opinions on using ( * ) for java import, but in this case, what can be the advantage?